app/integration_test/tests/sub_spaces.dart

```diff
@@ -42,8 +42,8 @@ void subSpaceTests() {
       await t.gotoSpace(spaceId);
       await t.navigateTo([
         const Key('spaces'),
-        SubSpacesPage.moreOptionKey,
-        SubSpacesPage.linkSubspaceKey,
+        SubSpaces.moreOptionKey,
```

Comment:
`SubSpacesPage` class I removed completely and created new class name `SubSpaces` so that git doesn't compare it with old code and easy to review new file code reviewer.

Reply:
I get the point but still I'd have it as `Page` widget so it's easier to differentiate between widgets that are route pages and UI components. We're following that pattern everywhere so this'd break consistency...

Reply:
I understand, generally I renamed it with `Page` right after this type of bigger PR. This is just for better git code review. Let me me renamed it.